x-pack/plugin/sql/qa/security/build.gradle

```diff
@@ -29,38 +29,23 @@ subprojects {
     testCompile project(":x-pack:plugin:core")
   }
 
-  integTestCluster {
+  testClusters.integTest {
+    distribution = "DEFAULT"
     // Setup auditing so we can use it in some tests
     setting 'xpack.security.audit.enabled', 'true'
     setting 'xpack.security.enabled', 'true'
     setting 'xpack.license.self_generated.type', 'trial'
     // Setup roles used by tests
-    extraConfigFile 'roles.yml', '../roles.yml'
+    extraConfigFile 'roles.yml', file('../roles.yml')
```

Comment:
I think this should just be `file('roles.yml')` since that file exists in the root of the project directory (peer of the build script). I suspect the relative path was needed before due to this being a simple string and how the test cluster working directory was set.

Reply:
that should have failed with a missing file exception as we do check for existence. 
We are in a `subprojects` block so `file` resolves to the subproject and thus needs to go to the parent. I;m rewriting this to make it more obvious.